src/wolfgame.py

```diff
@@ -6902,10 +6908,12 @@ def game_stats(cli, nick, chan, rest):
     if len(rest) and not rest[0].isdigit():
         gamemode = rest[0]
         if gamemode != "all" and gamemode not in var.GAME_MODES.keys():
-            gamemode, _ = complete_match(gamemode, var.GAME_MODES.keys())
-        if not gamemode:
-            cli.notice(nick, messages["invalid_mode_no_list"].format(rest[0]))
-            return
+            gamemode = complete_match(gamemode, var.GAME_MODES.keys())
```

Comment:
again, use a different variable name. A list isn't a gamemode, so don't call it that